go/oasis-node/cmd/stake/stake.go

```diff
@@ -90,6 +90,28 @@ func doWithRetries(cmd *cobra.Command, descr string, fn func() error) {
 	os.Exit(1)
 }
 
+func getTokenSymbol(ctx context.Context, cmd *cobra.Command, client api.Backend) string {
+	var symbol string
+	doWithRetries(cmd, "query token's symbol", func() error {
```

Comment:
What's up with these retries all over the staking cmds, these should never be needed. Can we open an issue to get rid of this and not introduce more in new code?

Reply:
Old code fixed in #3113. I've also removed references in this PR.